docs/configuration.md

```diff
@@ -1,6 +1,19 @@
 # Configuring source{d} Lookout
 
-**source{d} Lookout** is configured with the `config.yml` file, you can use the template [`config.yml.tpl`](/config.yml.tpl) to create your own. Use the `lookoutd` option `--config` to set the path to it, or use the default location at `./config.yml`.
+The behavior of **source{d} Lookout** is configured with two different configuration files:
+
+- [`config.yml`](#config.yml), to define the global configuration of the server.
```

Comment:
Link does not work, at least in GitHub, I don't know about Gitbook.
```suggestion
- [`config.yml`](#configyml), to define the global configuration of the server.
```

Reply:
I asumed GitBook as the preferred way to render our docs, and it works differently when creating anchors for titles having `.` inside.
- Title: `config.yml`
- GitBook: `#config-yml`
- GitHub: `#configyml`

In case it is required to support both, would you propose an alternative?

Reply:
For this PR it would be fine if it only works on GitBook, in my opinion. But if the section links do not work on GitHub let's at least open an issue and try to fix it at some point. Maybe we just need to assume the limitations and avoid `.` in titles.

Reply:
agree, issue created https://github.com/src-d/lookout/issues/469